src/shared/mkfs-util.h

```diff
@@ -18,6 +18,7 @@ int make_filesystem(
                 const char *root,
                 sd_id128_t uuid,
                 bool discard,
+                bool fsverity,
```

Comment:
So we typically have the rule that if a function has >= bool args, it should be turned into a proper flags field. This is qualified for that.

Reply:
... I was thinking the same, but was hoping to sneak under the radar ... :smirk:

Reply:
This is getting a bit ugly in terms of turning boolean values present at the callsite into the correct flags bits.

...how do you feel about multiplication of `bool` with `enum` values?

Reply:
fwiw, I just checked.  This outputs `6`:

```c
    bool b = 5;
    printf("%d\n", b * 6);

```

Reply:
So it's the difference between something like

```c
   (want_discard ? MKFS_DISCARD : 0) | (want_verity ? MKFS_FS_VERITY : 0)
```

and

```c
   want_discard * MKFS_DISCARD | want_verity * MKFS_FS_VERITY
```

I find the multiplication thing to be very deeply weird, and I'm kinda surprised to find myself proposing such a hack, but the fact that it is well-defined (even if the value assigned into the `bool` was not `1`/`0`) is pretty reassuring.  After I understand it, I find the second form much easier to read.

Note the operator precedence works with us in the case of `*` but against us with `?:`, since `*` > `|` > `?:`.

This is definitely a matter of taste: I let you guys decide.

Reply:
We always do the first form, let's stick to that here as well